code/game/machinery/doors/door.dm

```diff
@@ -402,7 +402,7 @@
 
 /obj/machinery/door/proc/CheckForMobs()
 	if(locate(/mob/living) in get_turf(src))
-		sleep(1)
+		sleep(1 TICKS)
```

Comment:
are you sure this wasn't for an animation? everything that uses is has animation sleeps.

Reply:
i didn't pay the closest of attention to what it called right afterwards.

Reply:
i can't be certain but i'm going to default to using seconds here because everything in the proc called right after it is in seconds